.typos.toml

```diff
@@ -0,0 +1,15 @@
+[default]
+extend-ignore-re = [
+    "Lorem ipsum [a-zA-Z .,]*",
+]
+
+[default.extend-identifiers]
+FillStrat = "FillStrat" # short for strategy
+
+[default.extend-words]
+seeked = "seeked" # FIXME?
```

Comment:
Where is `seeked` used?

Reply:
https://github.com/search?q=repo%3Alinebender%2Fxilem%20seeked&type=code

Reply:
It's part of the [HTML standard](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/seeked_event) so we should keep the typo to avoid confusion.